src/System.Management.Automation/Interop/Interop.HRESULT.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
I don't see a reason for making it more complex. I would prefer to keep the `Utils.Succeed`

Reply:
I don't see too. 😕